emr-dynamodb-hadoop/src/main/java/org/apache/hadoop/dynamodb/DynamoDBClient.java

```diff
@@ -229,10 +233,11 @@ public BatchWriteItemResult putBatch(String tableName, Map<String, AttributeValu
       writeBatchList = writeBatchMap.get(tableName);
     }
 
-    log.debug("BatchWriteItem deletionMode " + deletionMode);
+    log.info("BatchWriteItem deletionMode " + deletionMode);
```

Comment:
For information, this increases a lot the sizes of the log unless we limit our log levels to `WARN`. What was the motivation for this change?